daml/Marketplace/AssetDescription.daml

```diff
@@ -0,0 +1,33 @@
+module Marketplace.AssetDescription where
+-- Consider merging into DA.Finance.Types.Asset
+
+import DA.Finance.Types (Id)
+import ContingentClaims.Claim.Serializable (Claim)
+
+type T = AssetDescription
+type Claims = Claim Date Id
+
+template AssetDescription
+  with
+    assetId: Id
+    -- ^ This is the master record for the asset, and has embedded it's unique identifier
+    description: Text
+    -- ^ Short description of the asset, mainly for display purposes
+    issuer: Party
+    -- ^ The counterparty for any @claims@
+    claims: Claims
+    -- ^ Rights and obligations of the holder (and issuer)
+    safekeepingAccountId: Id
+    -- ^ Source account used for issuing the security and any subsequent corporate actions, e.g. paying out a coupon.
+    -- terms: Text
```

Comment:
Do you plan on enabling these commented out variables in the future ? If not, remove them.

Reply:
I'm not sure. @georg-da what's your take on this? Do you think the 'shares authorized' should be a property of the origination? It protects the investors from having the issuer simply mint more tokens and diluting holdings.

I remember thinking that there was another case where this was handy, but I've forgotten it now.

Reply:
I think we can revisit that topic at a later point. There needs to be a process by which an issuer sets or increases the authorized shares. Haven't thought enough about this to understand where that would fit in